tests/vmcheck/overlay.sh

```diff
@@ -59,12 +59,16 @@ fi
 cd /ostree/repo/tmp
 rm vmcheck -rf
 ostree checkout $commit vmcheck --fsync=0
+# https://github.com/projectatomic/rpm-ostree/pull/693
+rm -f vmcheck/usr/etc/{.pwd.lock,passwd-,group-,shadow-,gshadow-,subuid-,subgid-}
 rsync -rlv /var/roothome/sync/insttree/usr/ vmcheck/usr/
 # ✀✀✀ BEGIN hack for https://github.com/projectatomic/rpm-ostree/pull/642 ✀✀✀
 ostree admin unlock || true
-for url in https://kojipkgs.fedoraproject.org//packages/ostree/2017.2/3.fc25/x86_64/ostree-{,libs-,grub2-}2017.2-3.fc25.x86_64.rpm; do
-    curl -sSL -O $url
-done
+ver=2017.3
```

Comment:
Yeah, though I need the newer version for livefs.  Hmm.  I'll do something else for that.